tests/test_filepath.py

```diff
@@ -169,16 +164,9 @@ def test_filepath_cleanup(table=Filepath(), store="repo"):
     m = n - len(table)  # number deleted
     assert_true(m == 6)
 
-    ext.delete()  # delete unused entries
+    ext.delete(delete_external_files=True)  # delete unused entries
     assert_true(0 < len(ext) <= n - m)
 
-    unused_files = list(ext.get_untracked_filepaths())
-    assert_true(0 < len(unused_files) <= m)
-
-    # check no more untracked files
-    ext.clean_filepaths()
-    assert_false(bool(list(ext.get_untracked_filepaths())))
-
 
 def test_filepath_cleanup_s3():
     """test deletion of filepath entries from external table """
```

Comment:
This test failed on WIN10. For example, the result of `managed_file` was ` dj/repo/four\three\one\file.dat`.